comp/core/autodiscovery/listeners/common.go

```
@@ -68,18 +68,9 @@ func standardTagsDigest(labels map[string]string) string {
 
 // newContainerFilters instantiates the required container filters for AD listeners
 func newContainerFilters() (*containerFilters, error) {
-	global, err := containers.NewAutodiscoveryFilter(containers.GlobalFilter)
-	if err != nil {
-		return nil, err
-	}
-	metrics, err := containers.NewAutodiscoveryFilter(containers.MetricsFilter)
-	if err != nil {
-		return nil, err
-	}
-	logs, err := containers.NewAutodiscoveryFilter(containers.LogsFilter)
-	if err != nil {
-		return nil, err
-	}
+	global := containers.NewAutodiscoveryFilter(containers.GlobalFilter)
+	metrics := containers.NewAutodiscoveryFilter(containers.MetricsFilter)
+	logs := containers.NewAutodiscoveryFilter(containers.LogsFilter)
```

IMO,
we still need to return an error, 
but we should compute all the filter even if the first trigger an error. 
and then return an aggregated error

---

Discussed offline.

Conclusion is: no need to return an error here because the error is already reported in the agent status command output within the Autodiscovery section (see QA section for an example)